server.py

```diff
@@ -432,6 +432,32 @@ def put(self, region, id):
             tournament.regions = args['regions']
 
         dao.update_tournament(tournament)
+        return convert_tournament_to_response(dao.get_tournament_by_id(tournament.id), dao)
+
+    def delete(self, region, id):
+        """ Deletes a pending tournament.
+            If a (non-pending) tournament with this id exists, tell the user they can't do that.
```

Comment:
I think it's ok to open this up eventually to all tournaments instead of pending tournaments, but it's ok as is for now.